src/main/features/dashboard/views/defendant.njk

```diff
@@ -22,15 +22,11 @@
             {{ t('Sign the agreement') }}
           </a>
         </p>
-      {% elif featureToggles.offer and not claim.settlementReachedAt  %}
+        {% elif featureToggles.offer and claim.status === ClaimStatus.CLAIM_REJECTED  %}
```

Comment:
won't this block new offers?

Reply:
Will check that scenario

Reply:
From my discussion content designer, a defendant currently can only make one offer. In the future, there probably will be some functionality to deal with multiple offers. Hope this helps Tim

Reply:
I always thought that defendant can make as many offers as he would like as long as claimant rejected previous offer. In other words many concurrent offers should not be possible but many offers in single offer chain should be allowed.

Please check with business.

Reply:
on a closer look it looks right, i got mixed up with offer rejected.
but need to test with:
no offer made -> visible
offer made -> not visible
offer rejected -> visible
offer accepted -> not visible
countersigned -> not visible

Reply:
I tested this, It fixes the current bug,
I noticed another bug where rejecting an offer doesn't update what people can see.
i.e. claimant still gets 
> The defendant has made an offer to settle out of court.
View their offer 

defendant still get's
> You made an offer to settle out of court.

Then you get an error if you try reject again.
Hopefully its fixed in @varma02 and @geek-soft 's counterclaim PR

Reply:
@pragnesh-patel @vianvi FYI

Reply:
Yeah, rejecting an offer is not handled at all. When I merged counted claim story I will have closed look to make sure all the statuses are right and I will fix few more bugs if there is not story already for it.